apps/i18n/common/en_us.json

```diff
@@ -1065,6 +1065,7 @@
   "downloadUnitXHandouts": "Download all handouts for Unit {unitNumber}",
   "downloadUnitLessonPlans": "Download all lesson plans for unit",
   "downloadUnitHandouts": "Download all handouts for unit",
+  "dragAndDropInstructions": "To pick up a file, press m. While dragging, use the arrow keys or tab to move the file. Press m, space or enter to drop the file in its new position, or press escape to cancel.",
```

Comment:
should we say "pick up a file or folder" for the file browser instructions?

Reply:
also, would it make sense for this to be in `codebridge/en_us.json` instead?

Reply:
Happy to update to file 'or folder' ! And I thought about that- is codebridge the only place that will use this file system? I imagined we had built it in a way for it to be a 'lab 2' thing but you know more than I do about that!

Reply:
Sounds good! One question- now that I'm updating the first part to 'file or folder', should I switch the other 'file' references to 'item'?

Reply:
Probably? I think we should have 2 sets of instructions; one for the file tabs which can be this, and one for the file browser which references files and folders.

Reply:
Updated and updated screenshots!